src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/query/ia/CftQueryServiceAssignTaskTest.java

```diff
@@ -386,7 +386,7 @@ void should_return_empty_task_resource_when_authorisations_is_empty_for_challeng
     }
 
     private static Stream<GrantType> getGrantTypes() {
-        return Stream.of(GrantType.BASIC, GrantType.STANDARD, GrantType.SPECIFIC);
+        return Stream.of(GrantType.SPECIFIC, GrantType.STANDARD, GrantType.SPECIFIC);
```

Comment:
Is there any reason why GrantType.SPECIFIC is repeated ?

Reply:
BASIC and SPECIFIC types had the same functionality before this ticket. Therefore BASIC type were used when testing other issues. Since now BASIC type gets ignored, all those tests would be ignored too. To make sure tests run, I changed BASIC type to SPECIFIC type.